### ISSUE TITLE
where to continue "official" development

### ISSUE THREAD

**Issue by extrapixel** [01010101/GifAnimation#1]
Thanks a lot for porting! I suggest we:
- backport this repo to the original one
- release a new official version from there
- mention you alongside with me in the plugin description
- give you write permission to the original extrapixel repo

love to hear your opinion!

**Comment by 01010101** [comment]
Hi Patrick,

Of course, makes sense.
No hijacking intended but since I needed it for a class I'm teaching I did
those quick fixes and pushed all that to my Github to make it available to
my students and to people interested in the porting.

It's cool if you give me write permission. I'll push more examples too.

I'll change my repo so it reflects yours or at least point to yours.

++

J



Jerome Saint-Clair == *http://saint-clair.net <http://saint-clair.net/> || *
@_01010101 <https://twitter.com/_01010101>
Member of *http://graffitiresearchlab.fr <http://graffitiresearchlab.fr/>*

**Comment by extrapixel** [comment]
I just started merging. I saw you removed the getGifLength(). Is there a specific reason for that?

You removed setPalette() et al. This was an addition by @jimcavoli - but I'm not sure if it is still needed. 

Also the folder structure in the project (src folder) changed. Is it your personal preference or is there a specific reason?

It's a bit unfortunate that the port isn't a fork anymore in github terms - so it's all manual work.

Anyway, I added you as a collaborator.

**Comment by extrapixel** [comment]
let's continue discussion here: https://github.com/extrapixel/gif-animation/issues/11

**Comment by 01010101** [comment]
Hey,

Maybe I didn't start from the right source.
I'll dig into that when I have more time.
Most probably Wednesday next week.



Jerome Saint-Clair == *http://saint-clair.net <http://saint-clair.net/> || *
@_01010101 <https://twitter.com/_01010101>
Member of *http://graffitiresearchlab.fr <http://graffitiresearchlab.fr/>*

**Comment by extrapixel** [comment]
yes don't worry. I shoved it all into a 3.0 branch over at the original repo. Have not yet moved "reference" (need to discuss how to handle docs/reference) and reverted to the old folder structure. But we can diff now between the two forks (the old in master and the new in 3.0) to see what's missing and what's the differences.

**Comment by BLCKPSTV** [comment]
When will this be ready for the latest processing? :) Need it for working with another project. :)
